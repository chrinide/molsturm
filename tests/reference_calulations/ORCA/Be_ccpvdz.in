! bohrs hf EXTREMESCF nofrozencore nopop noprintmos cc-pvdz
* xyz 0 1
Be  0  0  0
*
