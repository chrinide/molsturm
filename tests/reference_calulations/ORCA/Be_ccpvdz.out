
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #  Department of molecular theory and spectroscopy    #
                  #              Directorship: Frank Neese              #
                  # Max Planck Institute for Chemical Energy Conversion #
                  #                  D-45470 Muelheim/Ruhr              #
                  #                       Germany                       #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.0.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Properties
   Michael Atanasov       : Ab Initio Ligand Field Theory
   Ute Becker             : Parallelization
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Dagmar Lenk            : GEPOL surface
   Dimitrios Liakos       : Extrapolation schemes; parallel MDCI
   Dimitrios Manganas     : ROCIS; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Restricted open shell CIS
   Masaaki Saitow         : Open-shell DLPNO
   Barbara Sandhoefer     : DKH picture change effects
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Georgi Stoychev        : AutoAux
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev                                     : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


Your calculation utilizes the basis: cc-pVDZ 
       H, B-Ne : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 T. H. Dunning, Jr., J. Chem. Phys. 90, 1007 (1989)
            He : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 D. E. Woon, T. H. Dunning, Jr., J. Chem. Phys. 100, 2975 (1994)
  Li-Be, Na-Mg : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 B. P. Prascher, D. E. Woon, K. A. Peterson, T. H. Dunning, Jr., A. K. Wilson, Theor. Chem. Acc. 128, 69 (2011)
         Al-Ar : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 D. E. Woon, T. H. Dunning, Jr., J. Chem. Phys. 98, 1358 (1993)
            Ca : Obtained from the Peterson Research Group Website (tyr0.chem.wsu.edu/~kipeters) Feb. 2017
                 J. Koput, K. A. Peterson, J. Phys. Chem. 106, 9595 (2002)
         Sc-Zn : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 N. B. Balabanov, K. A. Peterson, J. Chem. Phys. 123, 064107 (2005)
                 N. B. Balabanov, K. A. Peterson, J. Chem. Phys. 125, 074110 (2006)
         Ga-Kr : Obtained from the ccRepo (grant-hill.group.shef.ac.uk/ccrepo) Feb. 2017
                 A. K. Wilson, D. E. Woon, K. A. Peterson, T. H. Dunning, Jr., J. Chem. Phys. 110, 7667 (1999)

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================

Warning: TCutStore was < 0. Adjusted to Thresh (uncritical)

WARNING: TCut can not be >=Thresh
  ===> : TCut is set to 0.01*Thresh

INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = Be_ccpvdz.in
|  1> ! bohrs hf EXTREMESCF nofrozencore nopop noprintmos cc-pvdz
|  2> * xyz 0 1
|  3> Be  0  0  0
|  4> *
|  5> 
|  6>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  Be     0.000000    0.000000    0.000000

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 Be    4.0000    0     9.012    0.000000    0.000000    0.000000

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 Be     0   0   0     0.000000000000     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 Be     0   0   0     0.000000000000     0.00000000     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 1 groups of distinct atoms

 Group   1 Type Be  : 17s4p1d contracted to 3s2p1d pattern {881/31/1}

Atom   0Be   basis set group =>   1
------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   22
 # of primitive gaussian functions       ...   34
 # of contracted shells                  ...    6
 # of contracted basis functions         ...   14
 Highest angular momentum                ...    2
 Maximum contraction depth               ...    8
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  3.000e-16
 Primitive cut-off              TCut     ...  3.000e-18


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... Be_ccpvdz
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....    4
 Basis Dimension        Dim             ....   14
 Nuclear Repulsion      ENuc            ....      0.0000000000 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequeny         DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  3.000e-16 Eh
 Primitive CutOff       TCut            ....  3.000e-18 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... All-Criteria
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-14 Eh
 Max Density Change     TolMaxP         ....  1.000e-14
 RMS Density Change     TolRMSP         ....  1.000e-14
 Orbital Gradient       TolG            ....  1.000e-09
 Orbital Rotation angle TolX            ....  1.000e-09
 DIIS Error             TolErr          ....  1.000e-14


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 1.674e-01
Time for diagonalization                   ...    0.000 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.000 sec
Total time needed                          ...    0.000 sec

-------------------
DFT GRID GENERATION
-------------------

General Integration Accuracy     IntAcc      ...  4.340
Radial Grid Type                 RadialGrid  ... Gauss-Chebyshev
Angular Grid (max. acc.)         AngularGrid ... Lebedev-110
Angular grid pruning method      GridPruning ... 3 (G Style)
Weight generation scheme         WeightScheme... Becke
Basis function cutoff            BFCut       ...    3.0000e-16
Integration weight cutoff        WCut        ...    1.0000e-14
Grids for H and He will be reduced by one unit

# of grid points (after initial pruning)     ...   1678 (   0.0 sec)
# of grid points (after weights+screening)   ...   1678 (   0.0 sec)
Grid point division into batches done        ...    0.0 sec
Reduced shell lists constructed in    0.0 sec

Total number of grid points                  ...     1678
Total number of batches                      ...       27
Average number of points per batch           ...       62
Average number of grid points per atom       ...     1678
Average number of shells per batch           ...     5.79 (96.43%)
Average number of basis functions per batch  ...    13.50 (96.43%)
Average number of large shells per batch     ...     5.79 (100.00%)
Average number of large basis fcns per batch ...    13.50 (100.00%)
Maximum spatial batch extension              ...  21.83, 39.91, 39.91 au
Average spatial batch extension              ...  10.55, 12.37, 14.01 au

Time for grid setup =    0.008 sec

------------------------------
INITIAL GUESS: MODEL POTENTIAL
------------------------------
Loading Hartree-Fock densities                     ... done
Calculating cut-offs                               ... done
Setting up the integral package                    ... done
Initializing the effective Hamiltonian             ... done
Starting the Coulomb interaction                   ... done (   0.0 sec)
Reading the grid                                   ... done
Mapping shells                                     ... done
Starting the XC term evaluation                    ... done (   0.0 sec)
Transforming the Hamiltonian                       ... done (   0.0 sec)
Diagonalizing the Hamiltonian                      ... done (   0.0 sec)
Back transforming the eigenvectors                 ... done (   0.0 sec)
Now organizing SCF variables                       ... done
                      ------------------
                      INITIAL GUESS DONE (   0.1 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
  0    -14.5711751718   0.000000000000 0.01049882  0.00113070  0.0330820 0.7000
  1    -14.5715034819  -0.000328310102 0.00821431  0.00090131  0.0242464 0.7000
                               ***Turning on DIIS***
  2    -14.5717433611  -0.000239879232 0.02007137  0.00224209  0.0174788 0.0000
                      *** Initiating the SOSCF procedure ***
                           *** Shutting down DIIS ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  3    -14.57238080  -0.0006374403  0.001934  0.001934  0.002907  0.000301
               *** Restarting incremental Fock matrix formation ***
  4    -14.57233731   0.0000434908  0.000271  0.000446  0.001695  0.000158
  5    -14.57233761  -0.0000003027  0.000064  0.000138  0.000524  0.000049
  6    -14.57233763  -0.0000000177  0.000001  0.000000  0.000001  0.000000
  7    -14.57233763  -0.0000000000  0.000000  0.000000  0.000000  0.000000
                          ***Energy convergence achieved***
                           ***RMSP convergence achieved***
                           ***MAXP convergence achieved***
                       ***Gradient convergence achieved***
                    ***Orbital Rotation convergence achieved***
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   8 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -14.57233763 Eh            -396.53347 eV

Components:
Nuclear Repulsion  :            0.00000000 Eh               0.00000 eV
Electronic Energy  :          -14.57233763 Eh            -396.53347 eV
One Electron Energy:          -19.06194585 Eh            -518.70192 eV
Two Electron Energy:            4.48960822 Eh             122.16845 eV

Virial components:
Potential Energy   :          -29.14469121 Eh            -793.06737 eV
Kinetic Energy     :           14.57235358 Eh             396.53390 eV
Virial Ratio       :            1.99999891


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...    3.5527e-15  Tolerance :   1.0000e-14
  Last MAX-Density change    ...    0.0000e+00  Tolerance :   1.0000e-14
  Last RMS-Density change    ...    0.0000e+00  Tolerance :   1.0000e-14
  Last Orbital Gradient      ...    3.1336e-13  Tolerance :   1.0000e-09
  Last Orbital Rotation      ...    0.0000e+00  Tolerance :   1.0000e-09

             **** THE GBW FILE WAS UPDATED (Be_ccpvdz.gbw) ****
             **** DENSITY FILE WAS UPDATED (Be_ccpvdz.scfp.tmp) ****
             **** ENERGY FILE WAS UPDATED (Be_ccpvdz.en.tmp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000      -4.732326      -128.7731 
   1   2.0000      -0.309039        -8.4094 
   2   0.0000       0.058259         1.5853 
   3   0.0000       0.058259         1.5853 
   4   0.0000       0.058259         1.5853 
   5   0.0000       0.298823         8.1314 
   6   0.0000       0.350180         9.5289 
   7   0.0000       0.350180         9.5289 
   8   0.0000       0.350180         9.5289 
   9   0.0000       0.644274        17.5316 
  10   0.0000       0.644274        17.5316 
  11   0.0000       0.644274        17.5316 
  12   0.0000       0.644274        17.5316 
  13   0.0000       0.644274        17.5316 
--------------------------
ATOM BASIS FOR ELEMENT Be
--------------------------
 NewGTO Be
 S 9
    1       2940.000000000000         0.000679999806
    2        441.200000000000         0.005235998509
    3        100.500000000000         0.026605992426
    4         28.430000000000         0.099992971535
    5          9.169000000000         0.269701923223
    6          3.196000000000         0.451468871479
    7          1.159000000000         0.295073916000
    8          0.181100000000         0.012586996417
    9          0.058900000000        -0.003755966093
 S 16
    1       2940.000000000000        -0.000000000973
    2        441.200000000000        -0.000000007495
    3        100.500000000000        -0.000000038086
    4         28.430000000000        -0.000000143136
    5          9.169000000000        -0.000000386069
    6          3.196000000000        -0.000000646262
    7          1.159000000000        -0.000000422388
    8       2940.000000000000         0.000123000240
    9        441.200000000000         0.000966001888
   10        100.500000000000         0.004831009444
   11         28.430000000000         0.019314037755
   12          9.169000000000         0.053280104152
   13          3.196000000000         0.120723235990
   14          1.159000000000         0.133435260840
   15          0.181100000000        -0.530768037547
   16          0.058900000000        -0.580115671850
 end
-------------------------------------------
RADIAL EXPECTATION VALUES <R**-3> TO <R**3>
-------------------------------------------
   0 :     0.000000    27.721448     3.681853     0.414933     0.232865     0.165414
   1 :     0.000000     1.053068     0.522435     2.645763     8.380644    30.947925
   2 :     0.109926     0.150258     0.332100     3.716659    16.091040    78.294496
   3 :     0.109926     0.150258     0.332100     3.716659    16.091040    78.294496
   4 :     0.109926     0.150258     0.332100     3.716659    16.091040    78.294496
   5 :     0.000000     0.478817     0.395315     3.829802    17.866194    92.396228
   6 :     0.605143     0.455039     0.535270     3.071275    13.679642    73.989416
   7 :     0.605143     0.455039     0.535270     3.071275    13.679642    73.989416
   8 :     0.605143     0.455039     0.535270     3.071275    13.679642    73.989416
   9 :     0.097203     0.188320     0.412926     2.631218     7.434155   111.776461
  10 :     0.097203     0.188320     0.412926     2.631218     7.434155   111.776461
  11 :     0.097203     0.188320     0.412926     2.631218     7.434155   111.776461
  12 :     0.097203     0.188320     0.412926     2.631218     7.434155   111.776461
  13 :     0.097203     0.188320     0.412926     2.631218     7.434155   111.776461
-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.895 sec
Sum of individual times     ....       0.893 sec  ( 99.8%)

Fock matrix formation       ....       0.799 sec  ( 89.3%)
Diagonalization             ....       0.001 sec  (  0.1%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.000 sec  (  0.0%)
Initial guess               ....       0.085 sec  (  9.5%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.001 sec  (  0.1%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -14.572337630930
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... Be_ccpvdz.gbw
Electron density file                           ... Be_ccpvdz.scfp.tmp

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:      0.00000      -0.00000       0.00000
Nuclear contribution   :      0.00000       0.00000       0.00000
                        -----------------------------------------
Total Dipole Moment    :      0.00000      -0.00000       0.00000
                        -----------------------------------------
Magnitude (a.u.)       :      0.00000
Magnitude (Debye)      :      0.00000


Timings for individual modules:

Sum of individual times         ...        1.091 sec (=   0.018 min)
GTO integral calculation        ...        0.182 sec (=   0.003 min)  16.7 %
SCF iterations                  ...        0.908 sec (=   0.015 min)  83.3 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 1 seconds 182 msec
